openlibrary/templates/search/authors.html

```diff
@@ -59,7 +59,7 @@ <h1>
         $for doc in response['docs']:
             $ n = doc['name']
             $ num = doc['work_count']
-            $ wc = ungettext("1 book", "%(count)s books", num, count=commify(num))
+            $ wc = ungettext("1 book", "%(count)d books", num, count=num)
```

Comment:
I'm very uncomfortable with this change unless extensive review has been done. Normally I'd expect to see the messages.pot file and ALL language messages.po files revised as a result of this. Changes to parameter types break ALL translations.

Reply:
So as far as I can tell, po files only use the singular as the unique Id for the strong. That's what caused the bug here in the first place. Changing the plural form has no impact. Resolving as won't fix (not an issue). This is on Dev, so feel free to test as well and let me know if you notice issues 👍

Reply:
Or, more specifically, changing the plural won't have an impact in this case, since the po files had `%d` already (I checked them).